pulsar-client-cpp/lib/MultiTopicsConsumerImpl.h

```diff
@@ -134,11 +142,19 @@ class MultiTopicsConsumerImpl : public ConsumerImplBase,
     void handleOneTopicUnsubscribedAsync(Result result, std::shared_ptr<std::atomic<int>> consumerUnsubed,
                                          int numberPartitions, TopicNamePtr topicNamePtr,
                                          std::string& topicPartitionName, ResultCallback callback);
+    void runPartitionUpdateTask();
+    void topicPartitionUpdate();
+    void handleGetPartitions(const TopicNamePtr topicName, const Result result,
```

Comment:
```suggestion
    void handleGetPartitions(TopicNamePtr topicName, Result result,
```

Remove `const` for parameter that is neither a pointer (BTW, `std::shared_ptr` is a object, not a pointer) or a reference, see https://abseil.io/tips/109